base/src/main/java/com/smartdevicelink/proxy/rpc/ShowConstantTbt.java

```diff
@@ -55,8 +56,8 @@
     public static final String KEY_TEXT2 = "navigationText2";
     public static final String KEY_ETA = "eta";
     public static final String KEY_TOTAL_DISTANCE = "totalDistance";
-    public static final String KEY_MANEUVER_DISTANCE = "distanceToManeuver";
-    public static final String KEY_MANEUVER_DISTANCE_SCALE = "distanceToManeuverScale";
+    public static final String KEY_DISTANCE_TO_MANEUVER = "distanceToManeuver";
+    public static final String KEY_DISTANCE_TO_MANEUVER_SCALE = "distanceToManeuverScale";
```

Comment:
@kostyaBoss On line 60 I would rename this variable back to its old name `KEY_MANEUVER_DISTANCE_SCALE`.  You can apply this change throughout the class.